docs/book/src/reference/webhook-for-core-types.md

```diff
@@ -1,7 +1,7 @@
 # Admission Webhook for Core Types
 
 It is very easy to build admission webhooks for CRDs, which has been covered in
-the CronJob tutorial. Given that kubebuilder doesn't support webhook scaffolding
+the [CronJob tutorial](https://github.com/kubernetes-sigs/kubebuilder/tree/master/docs/book/src/cronjob-tutorial). Given that kubebuilder doesn't support webhook scaffolding
```

Comment:
Sure, I observed there are a lot of links that aren't within aliases tho

Reply:
Found something similiar 
https://github.com/kubernetes-sigs/kubebuilder/blob/3c8a6264f45edaaf9c2f1c9ce82ce7de26c16097/docs/book/src/multiversion-tutorial/tutorial.md?plain=1#LL11C63-L12C49

Reply:
Hi @lauchokyip,

Feel free to correct any links you come across that lack aliases. Ideally, they shouldn't exist.

Utilizing aliases and adhering to the documentation standards assist us in keeping the documents well-maintained.

Reply:
Also, if the following criteria for the docs are not in your Contributing guide, would be great get your help to add them:

To keep the maintainability of our docs, links must be:
- Added with alias: 

// TODO add example

- They should be added at the bottom of the doc
- If the link is for a Kubebuilder doc then, it should use relative paths instead of the URL

Reply:
> Hi @lauchokyip,
> 
> Feel free to correct any links you come across that lack aliases. Ideally, they shouldn't exist.
> 
> Utilizing aliases and adhering to the documentation standards assist us in keeping the documents well-maintained.

We could gather help from `good-first-issue` . I don't have the resources now but I can lead the effort when I have time

Reply:
> Also, if the following criteria for the docs are not in your Contributing guide, would be great get your help to add them:
> 
> To keep the maintainability of our docs, links must be:
> 
> * Added with alias:
> 
> // TODO add example
> 
> * They should be added at the bottom of the doc
> * If the link is for a Kubebuilder doc then, it should use relative paths instead of the URL

Sounds good, I think this should be in another PR since this title is "Added link for cronjob tutorial" , wdyt?

Reply:
HI @lauchokyip,

I think the most important is we ensure via reviews that we follow up that. 
But if someone would like to fix the places that we get the link merged without comply with the rule all fine.

Reply:
@camilamacedo86 , Do you mean by adding what you proposed into https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#how-to-contribute-to-docs? Sorry I don't quite understand what "ensure via reviews that we follow up that" mean?

I was thinking of also adding "For a smooth PR process, please ensure you read the CONTRIBUTING.md and that you include the following information:" and point people to the CONTRIBUTING guideline